src/editor.js

```diff
@@ -1,40 +1,40 @@
-import {EditorState, basicSetup} from "@codemirror/basic-setup"
-import {EditorView, keymap} from "@codemirror/view"
-import {indentWithTab} from "@codemirror/commands"
-import {javascript} from "@codemirror/lang-javascript"
+import { EditorState, basicSetup } from '@codemirror/basic-setup'
```

Comment:
I think we're moving to prefer double quotes over single quotes for strings, and formatted strings using backticks when values need to be injected.

Reply:
Hi, there is the change: 8fc4d42356672c6825a8b37e990d36a86505f77c.